your-first-app.md

```diff
@@ -0,0 +1,650 @@
+# Your first app
+
+Let's build a simple todo application using choo. This tutorial assumes you're
```

Comment:
We shouldn't use `simple` in here since it can potentially color someone's expectations of this works (they might not think what is happening is simple!).  Can we change this to `small`?